custodian/lobster/jobs.py

```diff
@@ -87,7 +87,8 @@ def run(self, directory="./"):
             zopen(os.path.join(directory, self.stderr_file), "w", buffering=1) as f_err,
         ):
             # use line buffering for stderr
-            return subprocess.Popen(cmd, cwd=directory, stdout=f_std, stderr=f_err)  # pylint: disable=R1732
+            cmd = " ".join(cmd)  # to join split commands (e.g., from atomate and atomate2)

```

Comment:
```suggestion
            if isinstance(cmd, list):
                cmd = " ".join(cmd)  # to join split commands (e.g., from atomate and atomate2)
```

We could also do this to make it safer in case a string is transferred or replace it with one of the shlex commands:
```suggestion
import shlex
cmd="test"
if isinstance(cmd, list):
    cmd = shlex.join(cmd) 
```